app.zen-browser.zen.yml

```diff
@@ -0,0 +1,55 @@
+app-id: app.zen-browser.zen
```

Comment:
app-id is invalid. As the doc mention, `-` is not allowed except in the last component.